README.md

```diff
@@ -1 +1,37 @@
-# code-runner
\ No newline at end of file
+Code Runner: An Environment to run your code
```

Comment:
**a tool to help you run your assignments**

Reply:
Done